apps/admin/backend/src/app.writeins.test.ts

```diff
@@ -190,11 +197,15 @@ test('write-in adjudication lifecycle', async () => {
 
     const writeInRecord = pendingWriteIn[0]!;
 
-    // get the ballot image data for the write in
-    const writeInImageEntries = await apiClient.getWriteInImage({
-      writeInId: writeInRecord.id,
-    });
-    expect(writeInImageEntries).toHaveLength(0); // the fixtures do not have ballot images
+    // Skipping this check on the write-in images. The fixture does now have
+    // write in images but attempting to retrieve them will throw an error
+    // because the endpoint is not compatible with VotingWorks format
+    // ballot layouts.
+
+    // const writeInImageEntries = await apiClient.getWriteInImage({
+    //   writeInId: writeInRecord.id,
+    // });
+    // expect(writeInImageEntries).toHaveLength(0); // the fixtures do not have ballot images
```

Comment:
Unless you have an eventual path to resolution, maybe just remove this whole chunk

Reply:
removed in a61f411710da5c024c74c8e0f96611bbac22b589